ui/inspectors/system-section.reel/overview.reel/overview.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <link rel="stylesheet" type="text/css" href="overview.css">
+    <script type="text/montage-serialization">
+    {
+        "owner": {
+            "properties": {
+                "element": {"#": "owner"}
+            },
+            "bindings": {
+                "allDatasets": {"<-": "@owner.object.datasets"},
+                "datasets": {"<-": "@owner.allDatasets.filter{volume == @owner.object.id}"}
+            }
+        },
+        "version": {
+            "prototype": "blue-shark/ui/field-text.reel",
+            "properties": {
+                "element": {"#": "version"},
+                "label": "System Version"
+            },
+            "bindings": {
+                "value": {"<-": "@owner.systemInfo.systemVersion"}
+            }
+        },
+        "hostname": {
+            "prototype": "blue-shark/ui/field-text.reel",
+            "properties": {
+                "element": {"#": "hostname"},
+                "label": "Host Name"
+            },
+            "bindings": {
+                "value": {"<-": "@owner.systemInfo.general.hostname"}
+            }
+        },
+        "dataset": {
+            "prototype": "blue-shark/ui/field-text.reel",
+            "properties": {
+                "element": {"#": "dataset"},
+                "label": "Total Datasets"
+            },
+            "bindings": {
+                "value": {"<-": "@owner.systemInfo.totalDatasets"}
+            }
+        },
+        "snapshots": {
+            "prototype": "blue-shark/ui/field-text.reel",
+            "properties": {
+                "element": {"#": "snapshots"},
+                "label": "Total Snapshots"
+            },
+            "bindings": {
+                "value": {"<-": "@owner.systemInfo.totalSnapshots"}
+            }
+        },
+        "shares": {
+            "prototype": "blue-shark/ui/field-text.reel",
+            "properties": {
+                "element": {"#": "shares"},
+                "label": "Total Shares"
+            },
+            "bindings": {
+                "value": {"<-": "@owner.systemInfo.totalShares"}
+            }
+        },
+        "peer": {
+            "prototype": "blue-shark/ui/field-text.reel",
+            "properties": {
+                "element": {"#": "peer"},
+                "label": "Total Peers"
+            },
+            "bindings": {
+                "value": {"<-": "@owner.systemInfo.totalPeers"}
+            }
+        },
+        "noOperation": {
```

Comment:
Why is this component named `noOperation` ? Something more explicit (like `dataTitle` or `dataSectionTitle`) would be better.